src/test/javascript/spec/mocks/mock-file-upload-submission.service.ts

```diff
@@ -0,0 +1,22 @@
+import { of } from 'rxjs';
+import { Participation, ParticipationType, StudentParticipation } from 'app/entities/participation';
+import { fileUploadExercise } from './mock-file-upload-exercise.service';
+import { EntityResponseType, FileUploadSubmission } from 'app/entities/file-upload-submission';
+import { Result } from 'app/entities/result';
+
+export const fileUploadParticipation = new StudentParticipation();
```

Comment:
The tests are fixed, but don't keep the objects in the scope of the MockService.
It would be better to e.g. provide a small factory to guarantee that a new object is created.
Otherwise this will likely cause issues again in the future.

E.g.

```
export const createFileUploadSubmission = () => {
  const fileUploadSubmission = new FileUploadSubmission();
  fileUploadSubmission.submitted = false;
  fileUploadSubmission.participation = fileUploadParticipation;
  fileUploadSubmission.id = 1;
  return fileUploadSubmission;
}
```

Reply:
Thank you for pointing this out. I didn't understand how it should work at first. Now it is changed